### PR TITLE
BXMSDOC-8260-main: Update docs from Thorntail to Quarkus

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc [hunk 1]
```diff
@@ -10,61 +10,86 @@ You can use the process instance migration (PIM) service to create, export and e
...
+    jdbc:
+      url: jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+    username: sa
+    password: sa
+  hibernate-orm:
+    database:
+      generation: update
+pim:
+  auth-method: file
+"%int":
```
Comment (ruromero) [inline doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc:73]: remove this from the docs as this profile is only used for integration tests

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc [hunk 2]
```diff
@@ -10,61 +10,86 @@ You can use the process instance migration (PIM) service to create, export and e
...
+  kieservers:
```
Comment (ruromero) [inline doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc:74]: You can leave the kieservers definition as the example

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc [hunk 3]
```diff
@@ -10,61 +10,86 @@ You can use the process instance migration (PIM) service to create, export and e
...
+    password: kieserver1!
 ----
 . Start the PIM service:
 +
 [source,subs="attributes+"]
 ----
 $ java -jar {PRODUCT_FILE}-process-migration-service-standalone.jar -s./config.yml
 ----
-. To enable auto-detection of a JDBC driver by Thorntail, add the `JAR` file of the JDBC driver name to the `thorntail.classpath` system property. For example:
+. To re-augment a Quarkus configuration you need to build your application with `quarkus.package.type=mutable-jar`:
```
Comment (ruromero) [inline doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc:85]: I'd add the reasons why someone would need to re-agument the build. The main reason would be to add more dependencies or to change build time properties as it is the example using the mariadb datasource
You may refer to the Quarkus documentation if needed (https://quarkus.io/guides/reaugmentation)

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc [hunk 4]
```diff
@@ -10,61 +10,86 @@ You can use the process instance migration (PIM) service to create, export and e
 . Download the `{PRODUCT_FILE}-add-ons.zip` file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page for {PRODUCT} {PRODUCT_VERSION}.
 . Unzip the downloaded archive.
 . Move the `{PRODUCT_FILE}-process-migration-service-standalone.jar` file from the add-ons archive to a desired location.
-. In the location, create a YAML file containing the kieserver and Thorntail configuration, for example:
+. In the location, create a YAML file containing the {KIE_VERSION} and Quarkus configuration, for example:
```
[the remark this answers is not in the document]
Comment (ruromero) [reply doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc:13]: I think it's worth adding the minimum requirements which are JRE 11+

**Review by ruromero — CHANGES_REQUESTED** [review]
Thanks @csherrar 
I've added some comments

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc:13]: Removed this step and added JRE 11+ to the requirements.

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc (hunk 3 above)
Comment (csherrar) [reply doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc:85]: Added the reasoning to the step and added a link the the relevant guide.

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc [hunk 5]
```diff
@@ -10,61 +10,86 @@ You can use the process instance migration (PIM) service to create, export and e
...
+    db-kind: h2
+    jdbc:
+      url: jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+    username: sa
+    password: sa
+  hibernate-orm:
+    database:
+      generation: update
+pim:
+  auth-method: file
```
[the remark this answers is not in the document]
Comment (csherrar) [reply doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc:72]: Added step.

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc (hunk 2 above)
Comment (csherrar) [reply doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc:74]: Removed the default code initially. How do you suggest it is structured in the procedure? @ruromero

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc:73]: Removed

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc (hunk 2 above)
Comment (ruromero) [reply doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc]: something like:
## Configure KIE Servers
In order to configure the connections to one or more KIE Servers, in the custom configuration file a list of `kieservers` must be defined:

```
kieservers:
  - host: http://kieserver1.example.com:8080/kie-server/services/rest/server
    username: admin
    password: admin123
  - host: https://kieserver2.example.com:8443/kie-server/services/rest/server
    username: foo
    password: bar
```

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc [hunk 6]
```diff
@@ -5,66 +5,36 @@ You can use the process instance migration (PIM) service to create, export and e
...
 +
-[source,subs="attributes+"]
 ----
-$ java -Dthorntail.classpath=./h2-1.4.200.jar -jar {PRODUCT_FILE}-process-migration-service-standalone.jar -s ./config.yml
+database:
+  generation: update
+pim:
+  auth-method: file
 ----
+. To add dependencies or modify the build time properties by re-augmenting your Quarkus application, build your application with the `quarkus.package.type=mutable-jar` option:
```
Comment (ruromero) [inline doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc:28]: this is already set in the base configuration

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc [hunk 7]
```diff
@@ -5,66 +5,36 @@ You can use the process instance migration (PIM) service to create, export and e
...
 ----
 $ java -jar {PRODUCT_FILE}-process-migration-service-standalone.jar -s./config.yml
 ----
-. To enable auto-detection of a JDBC driver by Thorntail, add the `JAR` file of the JDBC driver name to the `thorntail.classpath` system property. For example:
+. Ensure the relevant authentication method is set for `auth-method`. Valid values are `file`, `jdbc`, and `ldap`. For example:
 +
-[source,subs="attributes+"]
 ----
-$ java -Dthorntail.classpath=./h2-1.4.200.jar -jar {PRODUCT_FILE}-process-migration-service-standalone.jar -s ./config.yml
+database:
```
Comment (ruromero) [inline doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc:23]: If you're describing the `auth-method`s why add the `database.generation` property?
You can sue this example for LDAP:

```
pim:
  auth-method: ldap
quakus:
  security:
    ldap:
      dir-context:
        url: ldap://localhost:10389
        principal: uid=admin,ou=system
        password: secret
      identity-mapping:
        search-base-dn: ou=users,dc=example,dc=com
        rdn-identifier: uid
        attribute-mappings:
          from: uid
          filter: uniqueMember
          filter-base-dn: ou=groups,dc=example,dc=com

```

This one for file:

```
pim:
  auth-method: file
quarkus:
  security:
    users:
      file:
        enabled: true
        plain-text: true
        users: users.properties
        roles: roles.properties
```

And this one for Database:

```
pim:
  auth-method: jdbc
quarkus:
  jdbc:
    principal-query:
      sql: SELECT u.password, u.role FROM users u WHERE u.username=?
      clear-password-mapper:
        enabled: true
        password-index: 1
      attribute-mappings:
        0:
          index: 2
          to: groups
```

**Review by ruromero — CHANGES_REQUESTED** [review]

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc (hunk 2 above)
Comment (csherrar) [reply doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc]: Added

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc (hunk 6 above)
Comment (csherrar) [reply doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc]: Removed

**Review by ruromero — APPROVED** [review]
Looks good

**Pull request merged**